data/releases/schedule.yaml

```diff
@@ -128,69 +142,3 @@ schedules:
     - release: 1.25.0
       cherryPickDeadline: ""
       targetDate: 2022-08-23
-- release: 1.24
```

Comment:
Can you please add 1.24 to `eol.yaml`?

Reply:
sorry I forgot to commit that file! thanks for catching this.